cmd/nodecmd/status.go

```diff
@@ -46,9 +54,58 @@ func statusSubnet(_ *cobra.Command, args []string) error {
 		if blockchainID == ids.Empty {
 			return ErrNoBlockchainID
 		}
-		_, err = getNodeSubnetSyncStatus(blockchainID.String(), clusterName, true, false)
+		notSyncedNodes := []string{}
+		for _, host := range hostAliases {
+			isSubnetSynced, err := getNodeSubnetSyncStatus(blockchainID.String(), clusterName, host, true, false)
+			if err != nil {
+				return err
+			}
+			if !isSubnetSynced {
+				notSyncedNodes = append(notSyncedNodes, host)
+			}
+		}
+		printOutput(hostAliases, notSyncedNodes, clusterName, false)
+		return nil
+	}
+	notBootstrappedNodes, err := checkClusterIsBootstrapped(clusterName)
+	if err != nil {
 		return err
 	}
-	_, err := checkNodeIsBootstrapped(clusterName, true)
-	return err
+	printOutput(hostAliases, notBootstrappedNodes, clusterName, true)
+	return nil
+}
+
+func printOutput(hostAliases, notBootstrappedHosts []string, clusterName string, primaryNetwork bool) {
+	if len(notBootstrappedHosts) == 0 {
+		if primaryNetwork {
+			ux.Logger.PrintToUser(fmt.Sprintf("All nodes in cluster %s are bootstrapped to Primary Network!", clusterName))
+		} else {
+			ux.Logger.PrintToUser(fmt.Sprintf("All nodes in cluster %s are synced to Subnet %s", clusterName, subnetName))
+		}
+		return
+	}
+	ux.Logger.PrintToUser(fmt.Sprintf("Node(s) Status For Cluster %s", clusterName))
+	ux.Logger.PrintToUser("======================================")
+	for _, host := range hostAliases {
+		hostIsBootstrapped := true
+		for _, notBootstrappedHost := range notBootstrappedHosts {
+			if notBootstrappedHost == host {
+				hostIsBootstrapped = false
+			}
+			break
+		}
+		if hostIsBootstrapped {
+			if primaryNetwork {
+				ux.Logger.PrintToUser(fmt.Sprintf("Node %s is bootstrapped to Primary Network", host))
+			} else {
+				ux.Logger.PrintToUser(fmt.Sprintf("Node %s is synced to Subnet %s", host, subnetName))
+			}
+		} else {
+			if primaryNetwork {
+				ux.Logger.PrintToUser(fmt.Sprintf("Node %s is not bootstrapped to Primary Network", host))
+			} else {
+				ux.Logger.PrintToUser(fmt.Sprintf("Node %s is not synced to Subnet %s", host, subnetName))
+			}
+		}
```

Comment:
would not be nicer to have a table? with one nodes per row and node + primary + subnet as columns,
and OK/ERR or something like that on cells

Reply:
maybe an issue for this, or maybe make it here

Reply:
maybe colors to put emphasis on the problems